internal/compiler/passes/lower_popups.rs

```diff
@@ -76,11 +76,19 @@ fn lower_popup_window(
         popup_window_element.borrow_mut().base_type = window_type.clone();
     }
 
+    const CLOSE_POLICY: &str = "close-policy";
+    let close_policy = popup_window_element.borrow_mut().bindings.remove(CLOSE_POLICY);
+    let no_close_policy = close_policy.is_none();
+
     const CLOSE_ON_CLICK: &str = "close-on-click";
     let close_on_click = popup_window_element.borrow_mut().bindings.remove(CLOSE_ON_CLICK);
     let close_on_click = close_on_click
         .map(|b| {
             let b = b.into_inner();
+
+            if no_close_policy {
```

Comment:
If both `close-policy` and `close-on-click` are set, it should be an error, don't you think?